doc/nrf/includes/sample_board_rows.txt

```diff
@@ -221,3 +221,7 @@
 .. nrf54l15pdk_nrf54l15_cpuapp@soc1
 
 | :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` | | ``nrf54l15pdk_nrf54l15_cpuapp`` | ``nrf54l15pdk_nrf54l15_cpuapp@soc1`` |
```

Comment:
```suggestion
| :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` | | ``nrf54l15pdk_nrf54l15_cpuapp`` | ``nrf54l15pdk_nrf54l15_cpuapp@soc1`` |
```

Isn't this ref going to conflict with the one in line 227? I wondering this line shouldn't be removed, as soc1 is out of scope in sdk-nrf now.

Reply:
@annwoj @gmarull Any insights here?

Reply:
soc1 is not used upstream, so it's wrong